src/Storages/RabbitMQ/StorageRabbitMQ.h

```diff
@@ -77,6 +77,8 @@ class StorageRabbitMQ final: public IStorage, WithContext
     void incrementReader();
     void decrementReader();
 
+    LoggerPtr log;
```

Comment:
Have you thought about keeping `log` as a private member and pass it separately to the ctor of `RabbitMQSource`? I think it slightly better, because we don't expose anything and the source doesn't have to know which logger it should use. However I have no strong opinion on this, probably I wouldn't have commented on this would be the only comment.

Reply:
Yes, it is better not to move log to public, I was just being a bit lazy.